src/docfx/docset/Document.cs

```diff
@@ -100,6 +87,12 @@ internal class Document
 
         private readonly Lazy<(string docId, string versionIndependentId)> _id;
 
+        // TODO:
+        // This is a temporary property just so that legacy can access OutputPath,
+        // I'll slowly converge legacy into main build and remove this property eventually.
+        // Do not use this property in main build.
+        internal string OutputPath;
```

Comment:
>OutputPath [](start = 24, length = 10)

so in the future, legacy should get the `outputPath` from manifest, right?

Reply:
yes